lib/main_containers.pm

```diff
@@ -304,6 +304,9 @@ sub load_container_tests {
         if (!check_var('NETAVARK_BATS_SKIP', 'all')) {
             loadtest 'containers/netavark_integration' if (is_tumbleweed);
         }
+        if (!check_var('AARDVARK_BATS_SKIP', 'all')) {
+            loadtest 'containers/aardvark_integration' if (is_tumbleweed);
+        }
```

Comment:
sorry, but all this `skip` _inverted_ logic seems a bit unclear to me, if `skip` should mean `don't execute`: 
first, on top of all you enable all this _if-branch_ with `BATS_SKIP` **true**, that probably should mean `skip all` itself): what means then the _else_ case of `BAT_SKIP == false` that is emmpty ? 

Then, the other inner expressions `xxx_BATS_SKIP <> all` create a bit of confusion to me, sorry: wasn't it better to use a single parameter with many different values, to include a specific module, instead ?

Reply:
> sorry, but all this `skip` _inverted_ logic seems a bit unclear to me, if `skip` should mean `don't execute`: first, on top of all you enable all this _if-branch_ with a ` ...BATS_SKIP` **true**, that probably should mean `skip all` itself: what means then the _else_ case of `...BAT_SKIP == false` that is missing/empty ?

I don't use `true` because it's not a boolean expression.  No need for an `else` or defensive programming here in case it's missing or empty.

> Then, the other inner expressions `xxx_BATS_SKIP <> all` create a bit of confusion to me, sorry: wasn't it better to use a single parameter with many different values, to include a specific module, instead ?

We can't use a single variable for all modules.  The only case of use of a single variable is to run the whole set of upstream tests only if `PODMAN_BATS_SKIP` if set.

It's a pattern that has worked so far and it's documented here:

- https://confluence.suse.com/display/qasle/podman+upstream+tests
- https://confluence.suse.com/display/qasle/runc+upstream+tests
- https://confluence.suse.com/display/qasle/skopeo+upstream+tests

I plan to do a workshop on this to address any questions.